velox/type/tests/TypeTest.cpp

```diff
@@ -107,6 +107,61 @@ TEST(Type, TimestampComparison) {
   EXPECT_GE(t1, t1lessSeconds);
 }
 
+TEST(Type, Date) {
+  auto date0 = DATE();
```

Comment:
Naming variables a1, a2, a3... is a bad practice. Let's not do that. "date" should work.

Reply:
Understood. It would be nice to cleanup this file, but it is out of scope for this PR.